build.gradle.kts

```diff
@@ -783,13 +797,20 @@ tasks {
 
   register("copySubprojectDependencies", Copy::class) {
     subprojects.forEach() {
-      if (!it.name.startsWith("catalog") &&
-        !it.name.startsWith("authorization") &&
+      if (!it.name.startsWith("authorization") &&
+        !it.name.startsWith("catalog") &&
         !it.name.startsWith("cli") &&
-        !it.name.startsWith("client") && !it.name.startsWith("filesystem") && !it.name.startsWith("spark") && !it.name.startsWith("iceberg") && it.name != "trino-connector" &&
-        it.name != "integration-test" && it.name != "bundled-catalog" && !it.name.startsWith("flink") &&
-        it.name != "integration-test" && it.name != "hive-metastore-common" && !it.name.startsWith("flink") &&
-        it.parent?.name != "bundles" && it.name != "hadoop-common"
+        !it.name.startsWith("client") &&
+        !it.name.startsWith("filesystem") &&
+        !it.name.startsWith("flink") &&
+        !it.name.startsWith("iceberg") &&
+        !it.name.startsWith("spark") &&
+        it.name != "bundled-catalog" &&
```

Comment:
`bundled-catalog` is no longer in our code base, please remove it by the way.

Reply:
ok